### PR TITLE
reststops python

### PR DESCRIPTION
**Pull request by SansPapyrus683**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

File: solutions/usaco-810.mdx [hunk 1]
```diff
@@ -39,88 +41,81 @@ stop early, and our greedy algorithm is correct.
...
-			long long travelDist = x[i] - prevStopPos;
-			long long fTime = travelDist * fRate;
-			long long bTime = travelDist * bRate;
-			long long restTime = fTime - bTime;
-			ans += restTime * c[i];
-			prevStopPos = x[i];
+			long long travel_dist = x[i] - prev_stop_pos;
+			long long f_time = travel_dist * f_rate;
+			long long b_time = travel_dist * b_rate;
+			long long rest_time = f_time - b_time;
```
Comment (Aadit-Ambadkar) [inline solutions/usaco-810.mdx:90]: please change to ll

File: solutions/usaco-810.mdx [hunk 2]
```diff
@@ -152,13 +148,51 @@ public class RestStops {
 			}
 		}
 
-		BufferedWriter bw = new BufferedWriter(new FileWriter("reststops.out"));
-		bw.write(ans + "\n");
-		bw.close();
+		PrintWriter pw = new PrintWriter("reststops.out");
+		pw.println(ans);
+		pw.close();
```
Comment (Aadit-Ambadkar) [inline solutions/usaco-810.mdx:153]: we should use buffered writer...

File: solutions/usaco-810.mdx [hunk 3]
```diff
@@ -152,13 +148,51 @@ public class RestStops {
...
+with open('reststops.in') as read:
+	# trail_len won't be used
+	trail_len, stop_num, f_rate, b_rate = [int(i) for i in read.readline().split()]
+
+	x = []  # position of each stop
+	c = []  # tastiness value of each stop
+	for _ in range(stop_num):
+		a, b = [int(i) for i in read.readline().split()]
+		x.append(a)
+		c.append(b)
```
Comment (Aadit-Ambadkar) [inline solutions/usaco-810.mdx:171]:
```suggestion
with open('reststops.in') as fin:
	# trail_len won't be used
	trail_len, stop_num, f_rate, b_rate = [int(i) for i in fin.readline().split()]

	x = []  # position of each stop
	c = []  # tastiness value of each stop
	for _ in range(stop_num):
		a, b = [int(i) for i in fin.readline().split()]
		x.append(a)
		c.append(b)
```

**Review by Aadit-Ambadkar — CHANGES_REQUESTED** [review]

File: solutions/usaco-810.mdx (hunk 2 above)
Comment (SansPapyrus683) [reply solutions/usaco-810.mdx:153]: ?? like 90% of the sols here use pw or kattio

**Review by ryanchou-dev — APPROVED** [review]

**Pull request merged**